packages/nodes-base/nodes/Aws/S3/BucketDescription.ts

```diff
@@ -28,7 +28,7 @@ export const bucketOperations = [
 			{
 				name: 'Search',
 				value: 'search',
-				description: 'Search withim a bucket',
+				description: 'Search a bucket',
```

Comment:
@smamudhan Can you please change it to 'Search within a bucket', to make it consistent with the docs?